docs/connecting-to-test-db.md

```diff
@@ -0,0 +1,12 @@
+# Connecting to test environment database
+
+In order to connect to the test environment database you will need to be connected to the VPN. You will also need to source the `aws_helper_cyb.sh` script from the `configuarable-forms-infrastructure` repo and have `Kubernetes` and `Kubectl` installed.
```

Comment:
worth putting a link to this repo as well

Reply:
Good shout